src/app/components/ScrollablePromo/__snapshots__/index.test.jsx.snap

```diff
@@ -445,3 +445,284 @@ exports[`ScrollablePromo it should match a11y snapshot for single card 1`] = `
   </div>
 </div>
 `;
+
+exports[`recommendationEOJ it should match a11y snapshot for recommendationEOJ 1`] = `
```

Comment:
Is there a reason why we're using snapshots for this component? It would be preferable to create actual unit tests for checking a11y, instead of snapshots as they're not really useful when they fail as they're so big

Reply:
I don't think we do a11y testing in unit tests. The reason for this snapshot is to have a visible HTML for a11y purposes as we did previously for this component (A few tests above this).

Reply:
We shouldn't be treating a11y as a second class citizen, snapshots don't really provide any info. What exactly is the snapshot test testing a11y wise? When I look at the snapshot it's hard to figure out what it's testing. The react testing library we use recommends always unit testing for accessibility first https://testing-library.com/docs/dom-testing-library/api-accessibility/, so if there's a11y specific functionality/roles for this component, we should be using explicit unit tests instead of snapshots. I'm happy to pair on writing them if you want

Reply:
Correct me if I am wrong, I believed that all the a11y tests we do are either in e2e or integration tests. That is why I have said that we don't usually test for a11y in unit tests.
This test alone has no clear purpose and I agree. The last time we used snapshots to easily trace the DOM constructed by the component and to make sure that it followed the designs. I thought it could have been useful again.
I don't think there is any specific functionality/roles for this component that require any unit testing. 
Having said so, if there is any particular test that you have in mind that I need to address I am happy to add them 👍

Reply:
As for this test, it won't ever see a live or test environment

Reply:
We have a11y tests in e2e, which are the final "barrier" for catching a11y issues usine the axe library. We do carry out a11y unit testing in components e.g [CpsTopStories](https://github.com/bbc/simorgh/blob/815562ff4ceedb97bca648794d7e27a33d5aea30/src/app/containers/CpsTopStories/index.test.jsx#L128-L135). But if there's no specific a11y stuff for the component, can we just remove the snapshots entirely?

Reply:
Yes, as usual, my knowledge of Simorgh is full of flaws 😂. Thank you a lot Harvey, I will delete all these poor snapshots tests and add some proper a11y tests that I believe might be worth having.

Reply:
Not flaws at all haha, I still don't know half the stuff that Simorgh does. The PR is looking great, let me know if you need help with the a11y tests. This might be a good start as it will allow you to query elements by role https://testing-library.com/docs/queries/byrole, the example I linked to in CpsTopStories uses a document query selector, which we shouldn't really be using.